SparkyFitnessFrontend/src/api/Admin/backup.ts

```
@@ -26,3 +26,19 @@ export const restoreBackup = async (formData: FormData) => {
     isFormData: true,
   });
 };
+
+export const downloadLastBackup = async (): Promise<Blob> => {
+  const response = await fetch('/api/admin/backup/download', {
```

ESLint

[error] Unexpected use of 'fetch'. Native fetch is only allowed in src/api.

(no-restricted-globals)